pkg/mutation/mutators/modifyset/modify_set_mutator.go

```diff
@@ -77,7 +77,7 @@ func (m *Mutator) SchemaBindings() []runtimeschema.GroupVersionKind {
 	return m.bindings
 }
 
-func (m *Mutator) Value() (interface{}, error) {
+func (m *Mutator) Value(_ types.MetadataGetter) (interface{}, error) {
```

Comment:
Having to modify all of these types to satisfy the interface changes suggests we need to rethink the Mutator interface. Adding a new type of Mutator shouldn't immediately mean every other Mutator definition should change - that defeats the purpose of an interface.